x-pack/plugin/data-frame/src/main/java/org/elasticsearch/xpack/dataframe/transforms/DataFrameIndexer.java

```diff
@@ -180,7 +180,16 @@ protected void onFinish(ActionListener<Void> listener) {
 
     @Override
     protected SearchRequest buildSearchRequest() {
-        return pivot.buildSearchRequest(getPosition(), pageSize);
+        SearchRequest searchRequest = new SearchRequest(getConfig().getSource().getIndex());
```

Comment:
This code looks to functionally the same as 
`Pivot.buildSearchRequest(getConfig().getSource(), getPosition, pageSize)`

Reply:
yes, in this version. It is/will be fundamentally different for "continuous". 

I will re-check, you are probably right, in this branch I can call `pivot.build...`